lib/automatic_namespaces/autoloader.rb

```diff
@@ -27,7 +27,11 @@ def set_namespace_for_dir(pack_dir, package_namespace)
   end
 
   def pack_directories(pack_root_dir, metadata)
-    Dir.glob("#{pack_root_dir}/**/app/*").select { |dir| namespaced_directory?(dir, metadata) }
+    if metadata['root_autoload_path']
```

Comment:
how about this?

```
autoload_path = pack_root_dir
unless metadata['root_autoload_path']
  autoload_path += '/**/app/*'
end
Dir.glob("#{autoload_path}").select { |dir| namespaced_directory?(dir, metadata) }
```

Reply:
MM would you rather have the option be like `autoload_glob` which defaults to `**/app/*` then you could override it with

```
metadata:
  automatic_pack_namespace: true
  namespace_override: "Summer::Swim"
  autoload_glob: ''
```

Reply:
Actually, I do like that better... but i'll leave it to you to decide.

Reply:
Done - take one more look and then it's ready to merge.